libvast/test/system/type_registry.cpp

```diff
@@ -57,6 +68,15 @@ make_data_b(std::string a, vast::count b, vast::real c, std::string d) {
   return make_data(mock_layout_b, a, b, c, d);
 }
 
+// This one is incompatible with the others.
```

Comment:
In what sense incompatible?

Reply:
I've expanded on this a bit with the most recent commit (c6ce834)